Add docs about selecting web renderer at runtime (#5565)
flutter/website: src/docs/development/tools/web-renderers.md

This flag is ignored when a non-browser (mobile or desktop) device
target is selected.

## Runtime configuration

> Web renderer can be selected at runtime before the app is loaded:

Reviewer: ```suggestion
To override the web renderer at runtime:
```

> * Build the app with the `auto` (default) option.
> * Set `window.flutterWebRenderer` to `"canvaskit"` or `"html"` before the app
>   is loaded (e.g. before `main.dart.js` script).

Reviewer: ````suggestion
* Build the app with the `auto` option.
* Insert a `<script>` tag  in `web/index.html` file before the `main.dart.js`
  script.
* Set `window.flutterWebRenderer` to `"canvaskit"` or `"html"`:

```html
  <script type="text/javascript">
    let useHtml = // ...
    if(useHtml) {
      window.flutterWebRenderer = "html";
    } else {
      window.flutterWebRenderer = "canvaskit";
    }
  </script>
  <script src="main.dart.js" type="application/javascript"></script>
```
````

> Web renderer in Flutter is initialized at engine startup and can not be changed
> at runtime after that.

Reviewer: ```suggestion
The web renderer can't be changed after the Flutter engine startup process
begins in `main.dart.js`.
```
